src/tests/ftest/util/ssh_connection.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2019 Intel Corporation.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from __future__ import print_function
+
+from paramiko import client
+
+class Ssh(object):
+    """
+    It's mimic to the subprocess class, only difference is to use ssh protocol
+    and get command execution on remote machine where in subprocess command will
+    be initiated on local machine.
+    Right now check_output() and call() function implemented.In future more
+    methods can be added which will behave like subprocess methods.
+    """
+    def __init__(self, address, timeout=60, debug=False):
+        """
+        Initialize the remote machines and open SSH Connection.
+        Args:
+            address: remote machine IP address or hostname.
+            timeout: Timeout for waiting to finish the command
```

Comment:
Shouldn't `timeout` be an argument for the call() and check_output() functions?  I could see a default timeout being set inside init() in case a value isn't provided when running a command, but each command will vary in with which timeout should be used.